### DISCUSSION TITLE
Email from Google: "Specify a fixed SDK version to keep your app running on iOS 13"

### DISCUSSION THREAD

**Discussion by jarredt** [react-native-maps/react-native-maps#4398] [Q&A]
I received the following email from Google and wanted to confirm that I don't need to take specific action with my current version of `react-native-maps` (`0.31.1`). I am assuming not because I am specifically waiting to update to `>=1.0.0` because I am not ready to drop iOS 12 support... so I assume that I also don't need to worry about iOS 13 support getting dropped until I update to a future version of `react-native-maps`, and that this would be explicitly called out as a breaking change for that version?

> Hello Google Maps Customer,
> 
> We’re writing to let you know that starting Q2 of 2023 we will be freezing support for iOS 13 in an upcoming major version of Google Maps Platform SDKs for iOS.
> 
> What do I need to know?
> Versions of Google Maps Platform SDKs for iOS released beginning the second quarter of 2023 will support a minimum of iOS 14. Earlier SDK versions will continue supporting iOS 13.
> 
> If your dependencies do not specify a version number, your IDE will load the newest SDK version and new builds of your app will not support iOS 13.
> 
> Please read the Mobile OS version support policy for more information about this change.
> 
> What do I need to do?
> Specify a version (examples for Maps, Places) for the Google Maps Platform SDKs for iOS in your application's build dependencies to control when you raise the minimum supported OS for new versions of your app.
> 
> Our records show that some of your projects listed in the table below are used on devices running iOS 13. Please adjust your SDK dependencies in your next release to ensure continued support for them.
> 
> Project name (project number)	Package name	Impacted SDK	Usage percentage on iOS 13
> (redacted)	(redacted)	Maps SDK	0.25
> If you require additional information, read our blog posts about Understanding SDK Dependencies in Google Maps Platform.
> 
> If you have any questions or require assistance, please contact Google Maps Platform Support.
> 
> Thanks for choosing Google Maps Platform

**Comment by lailton-b** [comment]
I have the same doubt, were you able to get an answer somewhere?

**Reply by jarredt** [reply]
No I wasn't.

**Comment by DarckBlezzer** [comment]
What did you do?
